backend/src/journey/entity/legDetails.entity.ts

```diff
@@ -0,0 +1,67 @@
+import { ApiProperty } from "@nestjs/swagger";
+
+export class LegDetails {
+
+  @ApiProperty({
+    description: "Distance.",
+    type: Number
+  })
+  distance?: number;
+
+  @ApiProperty({
+    description: "Duration.",
+    type: Number
+  })
+  duration?: number;
+
+  @ApiProperty({
+    description: "Leg information.",
+    type: String
+  })
+  infos?: string; // Info[]
```

Comment:
What does this mean? If we dont have a better name for that, remove it and we add it back once we notice that we need it.